package.json

```diff
@@ -36,24 +36,24 @@
     "yarn.lock"
   ],
   "dependencies": {
-    "debug": "^4.3.1",
+    "debug": "^4.3.4",
     "passport-strategy": "*",
-    "q": "^1.5.0",
-    "xml-crypto": "^2.1.3",
+    "q": "^1.5.1",
+    "xml-crypto": "^3.0.0",
     "xml-encryption": "^2.0.0",
     "xml2js": "0.4.x",
     "xmlbuilder": "^15.1.1",
-    "@xmldom/xmldom": "0.7.4"
+    "@xmldom/xmldom": "0.8.3"
   },
   "devDependencies": {
-    "body-parser": "^1.17.1",
+    "body-parser": "^1.20.1",
     "eslint": "^7.32.0",
-    "express": "^4.16.2",
-    "mocha": "^9.1.4",
+    "express": "^4.18.2",
+    "mocha": "^9.2.2",
     "passport": "0.4.x",
-    "request": "^2.83.0",
+    "request": "^2.88.2",
     "should": "*",
-    "sinon": "^12.0.1"
+    "sinon": "^13.0.2"
   },
```

Comment:
`devDependencies` are mostly updated to the latest. Except:
- eslint, latest version dropped support for node 12
- mocha, latest version dropped support for node 12
- request, is now in the latest version, but is has critical issue which will not be patched. I'll try to patch vulnerable package in another commit. Axios could be potential alternative, but it will require test refactoring.
- passport, I'll try to update it, but it may require test refactoring.

Reply:
request -> json-schema issue is fixed by using `resolutions`.